lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -99,16 +110,22 @@ def update_user_whitelist(username, add=True):
 
         if options['add'] or options['del']:
             user_str = options['add'] or options['del']
-            add_to_whitelist = True if options['add'] else False  # pylint: disable=simplifiable-if-expression
+            enable = True if options['add'] else False  # pylint: disable=simplifiable-if-expression
+
             users_list = user_str.split(",")
             for username in users_list:
                 username = username.strip()
                 if username:
-                    update_user_whitelist(username, add=add_to_whitelist)
+                    try:
```

Comment:
I added this to keep the management command from blowing up in case we couldn't find the user from the command args. Previously, the command would stop executing after the `CommandError` was raised. This did not seem ideal.